src/renderer/utils/tootParser.js

```diff
@@ -71,7 +76,8 @@ export function parsePleromaAccount (accountURL) {
   const domainName = res[1]
   const accountName = res[2]
   return {
-    username: `@${accountName}`,
-    acct: `@${accountName}@${domainName}`
+    username: accountName,
+    acct: `${accountName}@${domainName}`,
```

Comment:
Why did you remove the leading `@` ?
Pleroma's account name does not contain `@`, so I think that it can not compare here: https://github.com/h3poteto/whalebird-desktop/pull/780/files#diff-77909af62ad381cc7fda2de882cccd56R50

Reply:
Apologies, I forgot to revert this change.